app/views/admin/editions/_show_metadata.html.erb

```diff
@@ -128,11 +128,7 @@
       <% end %>
     </h2>
     <% if @edition.attachments.any? %>
-      <% if @edition.imported? %>
-        <%= render partial: "edit_attachments", locals: { edition: @edition } %>
```

Comment:
I think this is the only place where `partial: "edit_attachments"` is rendered – so we should be able to delete the file [app/views/admin/editions/_edit_attachments.html.erb](https://github.com/alphagov/whitehall/blob/78e4f19186d2e7ba1ec0cdea7a657a996280646f/app/views/admin/editions/_edit_attachments.html.erb) 🗑️ 

Which means we should also be able to delete [app/assets/javascripts/admin_legacy/rename_attachments.js](https://github.com/alphagov/whitehall/blob/78e4f19186d2e7ba1ec0cdea7a657a996280646f/app/assets/javascripts/admin_legacy/rename_attachments.js) because it was only used by that partial. 💥